client/src/apps/Users/Users.module.css

```diff
@@ -0,0 +1,43 @@
+.container {
+  padding: 64px 30px;
+}
+
+.error {
+  display: block;
+  padding: 4px 0;
+  height: 20px;
+  color: red;
+}
+
+.tableWrapper {
+  overflow-x: scroll;
+  width: 100%;
+  max-height: 200px;
+  margin-bottom: 32px;
+}
+
+table {
+  background-color: #fff;
+}
+
+table {
+  border-collapse: collapse;
+  position: relative;
+  text-align: left;
+  white-space: nowrap;
+  width: 100%;
+}
+
+table > thead > tr > * {
+  background: silver;
```

Comment:
1. `thead` i tak semantycznie powinien być w `table`
2. Nie wiem czy w tym wypadku nie wystarczyłoby, żebyś te style ustawił osobno po prostu dla `thead` i dla `tbody`. Jeśli elementy wewnątrz nie nadpiszą tych styli to i tak odziedziczą je od rodzica. Chyba nawet z tym sticky powinno na luzie zadziałać. Wtedy cały element się przyklei (ale to już tak z głowy mówię :D)